geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -226,7 +226,7 @@ object Transformers extends EnhancedTokenParsers with LazyLogging {
       if (idx == -1) {
         idx = ctx.indexOf(n)
       }
-      ctx.get(idx)
+      if (idx < 0) null else ctx.get(idx)
```

Comment:
I can see how this would work if default and the "idx ==" check where both -2 instead of -1.  Then it would only do ctx.indexOf on the first eval() execution.